### ISSUE TITLE
Improper warning for aerialway=zip_line

### ISSUE THREAD

**Issue by Schroecker** [osm-fr/osmose-backend#2068]
item/class 9001001
missing tag   usage without railway or waterway
i.E. for way 604738413
Please complement "usage" for aerialway=zip_line

**Comment by Famlam** [comment]
Hello,

This validation warning originates from JOSM (specifically, [this part](https://josm.openstreetmap.de/browser/josm/trunk/resources/data/validator/combinations.mapcss#L558) of their code). We import these rules from JOSM into Osmose.
Could you please report it to them? You can report it here: https://josm.openstreetmap.de/newticket

In order to stop this warning, they should change
```
way[usage][!railway][!waterway][route!=railway][man_made!=pipeline] {
```
to 
```
way[usage][!railway][!waterway][route!=railway][man_made!=pipeline][!aerialway] {
```


In addition, although the use of `usage=*` is documented on the wikis of [Key:aerialway](https://wiki.openstreetmap.org/wiki/Key:aerialway), [Tag:aerialway=zip_line](https://wiki.openstreetmap.org/wiki/Tag%3Aaerialway%3Dzip_line) and several others for many years with the intention to deprecate the unspecific tag [aerialway=goods](https://wiki.openstreetmap.org/wiki/Tag%3Aaerialway%3Dgoods), it is not documented on the wiki of [Key:usage](https://wiki.openstreetmap.org/wiki/Key:usage) itself

**Comment by Famlam** [comment]
Thank you for filing https://josm.openstreetmap.de/ticket/23298 !

**Comment by tsmock** [comment]
In [19165/josm](https://josm.openstreetmap.de/changeset/19165/josm):

Fix [#23298](https://josm.openstreetmap.de/ticket/23298): Improper warning for aerialway=zip_line

Exclude aerialway from combinations check and add usage values to aerialway
presets (where someone has already used the key with the aerialway tag).

**Comment by Famlam** [comment]
Deployed, thanks @tsmock
